pkg/sql/opt/testutils/opttester/opt_tester.go

```diff
@@ -661,14 +663,14 @@ func fillInLazyProps(e opt.Expr) {
 		norm.DerivePruneCols(rel)
 
 		// Derive columns that are candidates for null rejection.
-		norm.DeriveRejectNullCols(rel)
+		norm.DeriveRejectNullCols(md, rel)
```

Comment:
I'm open to better ideas on how to pipe the metadata here, if you folks have any.